### ISSUE TITLE
Click to load Disus comments scriptlet not working 

### ISSUE THREAD

**Issue by troysjanda** [uBlockOrigin/uAssets#6515]
### URL(s) where the issue occurs
https://betanews.com

### Describe the issue
Click to load Disqus comments are not working, button is there but when clicked does not unblock comment section.

### Screenshot(s)

N/A (will not show in image the click or loading/ not loading process)

### Versions

- Browser/version: FF 70.0.1
- uBlock Origin version: 1.23.1b14
- Nano Defender

### Settings
All Ubo lists, Easylist/privacy 

All marked under setting/privacy

### Notes: conversation with gwarser on Ubo Subreddit

> Something broken in engine. @@||disqus.com/embed.js?_=1457540*$script is not matching in the middle of longer number. (In https://betanews.disqus.com/embed.js?_=1457540701460 for ex.)

> $ git bisect good && ./tools/make-firefox.sh 
> 7971b223855d3afae2ca57565f27da0631a1b045 is the first bad commit
> commit 7971b223855d3afae2ca57565f27da0631a1b045
> Author: Raymond Hill <rhill@raymondhill.net>
> Date:   Mon Oct 21 08:15:58 2019 -0400
> 
>     Expand bidi-trie usage in static network filtering engine
> 
>     Related issues:...

This was his workaround for the time being

> Workaround: add @@||betanews.disqus.com/embed.js?_=$script to My filters. [edit: or @@||disqus.com/embed.js?_=$script in general]
> 
> Can you create new issue on uBlock-issues? (copy my comments to it, I'm too sleepy)

**Comment by okiehsch** [comment]
> Click to load Disqus comments are not working, button is there but when clicked does not unblock comment section.

I can't reproduce.
![image](https://user-images.githubusercontent.com/16838044/68096761-47e06480-feb3-11e9-900d-571d47450974.png)


What is blocking `betanews.disqus.com/embed.js` on your end?

**Comment by gwarser** [comment]
Fixed by https://github.com/gorhill/uBlock/commit/7c0294bd5f541951c45f3841cfad7dddfd52be4d

Thankyou!

**Comment by troysjanda** [comment]
there is a scriptlet that makes the comment section not load until the button is clicked. It is this that is broken. gwarser has also found it to be broken as outlined in his comments

> Something broken in engine. @@||disqus.com/embed.js?_=1457540*$script is not matching in the middle of longer number. (In https://betanews.disqus.com/embed.js?_=1457540701460 for ex.)

I found the issue on Betanews but is broken on every site that uses Disqus comment system. his work around fixes the issue both the specific Betanews filter and the more broad filter that fixes it for other sites.

> Workaround: add @@||**betanews.disqus.com**/embed.js?=$script to My filters. [edit: or @@||disqus.com/embed.js?=$script in general]

**Comment by okiehsch** [comment]
> there is a scriptlet that makes the comment section not load until the button is clicked.

As you can see fromm my screenshot the comment section loads on my end with a default setup.
The filter
`@@||disqus.com/embed.js?=$script` 
does not do anything because the request `disqus.com/embed.js` is not blocked in the first place on my end.
Hence me asking you what is blocking that request on your end.

**Comment by okiehsch** [comment]
![image](https://user-images.githubusercontent.com/16838044/68097552-ea9ae200-feb7-11e9-8f95-6bd42f4354e5.png)

If you open the logger and search for that request what do you see?

**Comment by troysjanda** [comment]
it was **fixed** so it was broken at some point @okiehsch Closed

**Comment by okiehsch** [comment]
> Yes and it will by defaut display the comment section but when you add the following filters

Thanks for the clarification.
